docker/lightgbm-ci-build-env-alpine/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM alpine:3.18.4
+
+RUN apk add clang git cmake make gcc g++ tar zip curl openjdk8 bash
```

Comment:
is openmp being installed and used?

Reply:
I think openmp in being installed alongside g++. Since it also installs gcc v12.2.1 and according to this [gcc.gnu.org/wiki/openmp](https://gcc.gnu.org/wiki/openmp), it fully supports OpenMP 4.5.